addons/website_sale/models/res_config_settings.py

```diff
@@ -45,7 +45,12 @@ def _default_recovery_mail_template(self):
         default=_default_order_mail_template, domain="[('model', '=', 'sale.order')]",
         help="Email sent to customer at the end of the checkout process")
 
-    automatic_invoice = fields.Boolean("Automatic Invoice")
+    automatic_invoice = fields.Boolean("Automatic Invoice",
+                                       help="""The invoice is generated automatically and available in the customer portal
+                                       when the transaction is confirmed by the payment acquirer.
```

Comment:
This is the equivalent to 
```python
help="The invoice is generated automatically and available in the customer portal\n                                    when the transaction is confir...`
```
You can to split your text on multiple lines but do not make it a multiline content.
You can use one of https://www.smallsurething.com/multi-line-strings-in-python/